stardog/content.py

```diff
@@ -214,6 +218,9 @@ def __init__(
         self.content_encoding = content_encoding if content_encoding else c_enc
         self.input_type = input_type if input_type else c_input_type
         self.separator = separator if separator else c_separator
+        self.iri = (
+            iri if iri else f"file://{name}" if name else f"stream://{uuid.uuid4()}"
```

Comment:
does `stream://` ever need to be used?

Reply:
We do not need to use `stream://` but we need to create an IRI. This is the IRI format used by the server.